docs/source/index.md

```diff
@@ -214,9 +214,9 @@ pluginManagement {
 ```
 And then use the `4.0.0-beta.5-SNAPSHOT` version for the plugin and libraries.
 
-### Android Studio / IntelliJ plugin
+These snapshots are updated on each push to `main`.
 
-Snapshots for the Android Studio / IntelliJ plugin [are also available](tree/main/intellij-plugin#snapshots). 
+Weekly snapshots for the Android Studio / IntelliJ plugin [are also available](tree/main/intellij-plugin#snapshots). 
```

Comment:
Was already incorrect, the link needs to point to GH from the docs site.
```suggestion
Weekly snapshots for the Android Studio / IntelliJ plugin [are also available](https://github.com/apollographql/apollo-kotlin/tree/main/intellij-plugin#weekly-snapshots).
```

Reply:
Do we want to add/move this to the docs? 

Traditionally, the docs are a superset of GH. We only kept the `README.md` to avoid an extra hoop for quickstart but It'd probably makes sense to have this info in the docs as well?

Reply:
Actually we do have it in the docs! Here → https://www.apollographql.com/docs/kotlin/v4/testing/android-studio-plugin#weekly-snapshots

So I guess:

```suggestion
Weekly snapshots for the Android Studio / IntelliJ plugin [are also available](v4/testing/android-studio-plugin#weekly-snapshots). 
```
?

Reply:
Perfect 👍